integreat_cms/static/src/css/style.scss

```diff
@@ -53,6 +53,7 @@ header {
 nav#primary-navigation {
   #menu {
     max-height: calc(100vh - 50px);
+    padding-bottom: 2.5rem;
```

Comment:
```suggestion
```
I would put this into the template by changing `pb-2` to `pb-10` here:
https://github.com/digitalfabrik/integreat-cms/blob/33a4ccf6fb2734fd2f89f077a045136cae88b2d6/integreat_cms/cms/templates/_base.html#L99